doc/source/changelog.rst

```diff
@@ -2,6 +2,62 @@
 Changelog
 #########
 
+Schema Release `v0.6.0a10`
+==========================
+
+Changes to ``ga4gh/schemas`` ``master`` branch since version ``0.6.0a9`` (Jan 23, 2016)
+
+Features:
+ * Remove feature_id from ExpressionLevel and addeed ability to seearch by Name field. #818  Impacts
+  * `POST /expressionlevels/search`
+  * `GET /expressionlevels/{id}`
+ - Added support for BigWig files in a new Continuous Data object #802 Adds the following new endpoints:
+  - Continuous Data: `POST /continuoussets/search`
+  - Continuous Data: `GET /continuoussets/{id}`
+  - Continuous Data: `POST /continuous/search`
+ * Add deep set/get attr to protocol module #816
+ * Changed ontology term “id” to “term_id” #805 Impacts the following message types:
+  * OntologyTerm
+ * Replaced info fields with rich type Attributes fields #700 Impacts the following message types:
+  * TranscriptEffect
```

Comment:
I think that they are easier to read in list form, so I would like to leave them this way.